packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js

```diff
@@ -155,6 +155,9 @@ describe('validation', function() {
     testTemplate('element-type');
 
 
+    testOnly('element-type-no-value');
```

Comment:
```suggestion
    testTemplate('element-type-no-value');
```

Reply:
We could have a lint rule for this, in the future 😆